app/models/user.rb

```diff
@@ -1,8 +1,12 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
```

Comment:
Remove the duplicate devise line, we added this when we generated it with activeadmin but it's wrong